modules/programs/darcs.nix

```
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.programs.darcs;
+in {
+  meta.maintainers = with maintainers; [ chris-martin ];
+
+  options = {
+    programs.darcs = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "If enabled, install Darcs and config files.";
+      };
+      package = mkOption {
+        type = types.package;
+        default = pkgs.darcs;
+        defaultText = literalExpression "pkgs.darcs";
+        description = "The Darcs package to use.";
+      };
```

Could use `mkPackageOption`:

```suggestion
      package = mkPackageOption pkgs "darcs" { };
```